lib/elixir/src/elixir_parser.yrl

```diff
@@ -619,7 +619,10 @@ meta_from_location({Line, Column, EndColumn})
 %% Operators
 
 build_op({_Kind, Location, 'in'}, {UOp, _, [Left]}, Right) when ?rearrange_uop(UOp) ->
+  %% TODO: Deprecate `not left in right` rearrangement
```

Comment:
It'd better to replace backticks with regular double quotes, and we usually have period in comments if they are not titles.